site/docs/4.3/components/forms.md

```diff
@@ -1075,21 +1075,21 @@ If your form layout allows it, you can swap the `.{valid|invalid}-feedback` clas
   <div class="form-row">
     <div class="col-md-6 mb-3">
       <label for="validationTooltip03">City</label>
-      <input type="text" class="form-control" id="validationTooltip03" placeholder="City" required>
+      <input type="text" class="form-control" id="validationTooltip03" required>
       <div class="invalid-tooltip">
         Please provide a valid city.
       </div>
     </div>
     <div class="col-md-3 mb-3">
       <label for="validationTooltip04">State</label>
-      <input type="text" class="form-control" id="validationTooltip04" placeholder="State" required>
+      <input type="text" class="form-control" id="validationTooltip04" placeholder="Choose..." required>
```

Comment:
"choose..." as placeholder for a text input doesn't make sense. for consistency with the previous form earlier in the page, suggest replacing this with an actual `<select>` instead

Reply:
These 4 have been changed to `<select>`.